modules/util/source-formatter/src/main/resources/sourcechecks.xml

```diff
@@ -1633,6 +1633,10 @@
 			<category name="Upgrade" />
 			<description name="Run code migration to replace '@ServiceReference' by '@Reference'" />
 		</check>
+		<check name="UpgradeJavaThreadPoolExecutorClassCheck">
+			<category name="Upgrade" />
+			<description name="Run code to add ThreadPoolExecutor when instantiating CallerRunsPolicy()" />
```

Comment:
@TamyrisTorres, no need of parenthesis in `CallerRunsPolicy()`, please remove them :wink: